rib/rib.go

```diff
@@ -1725,6 +1732,7 @@ func (r *RIBHolder) Flush() error {
 			backupNHGs = append(backupNHGs, *nhg.BackupNextHopGroup)
 		}
 	}
+	// todo: xw-g check how backup NHG reference is counted.
```

Comment:
```suggestion
	// TODO(xw-g): check how backup NHG reference is counted.
```

I'm not sure that we have handling for backup NHG at all right now.